config/application.rb

```diff
@@ -31,10 +31,12 @@ class Application < Rails::Application
     # encrypted credentials, it's not appropriate for an open-source project
     if File.exist?(Rails.root.join('config', 'secrets.yml'))
       config.secrets = config_for(:secrets) # loads from config/secrets.yml
-      config.secret_key_base = config.secrets[:secret_key_base]
+      if config.secrets.present?
```

Comment:
Needed to get a deployment working.  I think, because there is a mis-configuration on the infrastructure side.